nrf_802154_sl/README.rst

```diff
@@ -16,22 +23,35 @@ Currently, the library implements the following features:
 * Timestamping.
   It provides precise frame timestamps.
 
-The library also defines APIs for features that are not yet supported:
+It also provides an API to retrieve the capabilities of the binary in run-time.
+
+Moreover, the library also defines APIs for features that are not yet supported, as the following:
 
 * WiFi coexistence support.
   It provides an API for facilitating the integration with an external PTA.
 * Antenna diversity support.
   It provides an API for runtime antenna selection.
 
-The library is available as binaries for the following SoCs:
+Requirements
+************
+
+The service layer is available as a binary for the following SoCs/SiPs:
 
-* nRF52840
-* nRF52833
-* nRF52820
 * nRF52811
+* nRF52820
+* nRF52833
+* nRF52840
+* nRF5340 Network Core
+
+.. note::
+
+   Currently, only multiprotocol support and CSMA/CA support are available for the nRF5340 Network Core.
```

Comment:
I think it's not valid anymore. It was fixed for 1.5

Reply:
Are you sure? The remaining features such as timestamping, delayed TRX etc. are not supported for nRF5340 at the moment.

Reply:
Right. I've misunderstood